Docs grammar and typos check (#6064)
vuestorefront/vue-storefront: packages/core/docs/enterprise/feature-list.md

**1. No UI/functional limitations**

> Vue Storefront is just a framework, and you can build your UI in any way you want. Anything that is possible with Nuxt is also possible with Vue Storefront

Reviewer: ```suggestion
Vue Storefront is just a framework, and you can build your UI in any way you want. Anything possible with Nuxt is also possible with Vue Storefront.
```

**2. Backend-agnostic**

We have the same interfaces for the same types of vendors (eCommerce/Search/CMS/...). Because of that, you can keep most of your frontend unaware of the backend services. If you decide to change any part of your eCommerce stack in the future, your frontend could remain untouched.

**3. Focus on performance**

> We're providing a bulletproof architecture, great defaults, best practices, and tools to make sure your app will perform smoothly. The majority of business logic is handled on the server side to make sure client bundles are small.

Reviewer: ```suggestion
We're providing a bulletproof architecture, great defaults, best practices, and tools to make sure your app will perform smoothly. The majority of business logic is handled on the server-side to make sure client bundles are small.
```

**4. No vendor lock-in**

You own your code, so it's up to you what services you will use for eCommerce or CMS. You decide where and how you will host it and deploy - Vue Storefront will never lock you into its services.

**5. Focus on quick time to market**

> We've built Vue Storefront to make eCommerce frontends development process as enjoyable and smooth as possible. Just like Vue and Nuxt themselves, we're focusing on getting things done, providing good defaults, and giving our users integrations and extensions they can use to speed up their development process.

Reviewer: ```suggestion
We've built Vue Storefront to make the eCommerce frontends development process as enjoyable and smooth as possible. Just like Vue and Nuxt themselves, we're focusing on getting things done, providing good defaults, and giving our users integrations and extensions they can use to speed up their development process.
```

**6. All-in-one modular solution.**

Vue Storefront is a set of tools that can work together (or independently) to support you in all areas of development. We provide out-of-the-box integrations, a framework for connecting all of them, a library of UI components, and a cloud hosting service.

You can let us take care of all aspects of your eCommerce presence or replace any of those parts with your own, custom services.

**7. Huge and vibrant community**

> Vue Storefront is the only truly open source eCommerce frontend framework with an active, and vibrant community of thousands developers all around the world. If you choose Vue Storefront, you're never left on your own.

Reviewer: ```suggestion
Vue Storefront is the only truly open-source eCommerce frontend framework with an active and vibrant community of thousands of developers worldwide. If you choose Vue Storefront, you're never left on your own.
```
